gpii/node_modules/processReporter/src/processesBridge.js

```diff
@@ -77,7 +77,7 @@ fluid.defaults("gpii.processes", {
         // Contesxt aware invokers.
         getProcessList: {
             funcName: "gpii.processes.get",
-            args: ["{that}.getPlatformProcesssList", "{arguments).0"]
+            args: ["{that}.getPlatformProcesssList", "{arguments}.0"]
```

Comment:
As well as the "Processs" typo this invoker (with the distinction from getPlatformProcessList) is unncessary. We can do with just one invoker and have the platform-specific impl override it in place. Note various other typos - "commadn", "identifeer", "Contesxt", etc.

Reply:
Okay, done.  Also, I propagated the change to the windows and linux pull requests.

Thanks.